src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -313,6 +316,10 @@ def _monitor_workers_and_process_tasks_in_thread(
                 # Handle process crashed.
                 if crashed:
                     bulk_logger.warning(f"Process crashed while executing line {line_number}.")
+                    LogName_i = "{}_{}.log".format(LogName, index)
+                    with open(os.path.join(LogPath, LogName_i), "r") as f:
```

Comment:
Need to handle the case where the file does not exist.

Reply:
fixed